contributing-device-support.md

```diff
@@ -1,19 +1,25 @@
-# Custom Board Support
+# Customer Board Support
+
+There are several options for onboarding the hardware onto balenaOS:
+
+1. You can find another board that would work for your case that is supported in our [supported device list](https://www.balena.io/docs/reference/hardware/devices/).
```

Comment:
```suggestion
1. Checking out if balenaOS doesn't already provide support for your board through our [supported devices list](https://www.balena.io/docs/reference/hardware/devices/).
```

This guide is probably in a place where they are checking out the custom image build process. So this point can be better phrased as a check that they should make.